cider/tests/nextgen/jitlib/JITLibTest.cpp

```diff
@@ -143,6 +146,43 @@ TEST_F(JITLibTests, LogicalOpTest) {
       false, true, true, [](JITValue& a, JITValue& b) { return !a; });
 }
 
+TEST_F(JITLibTests, TupleTest) {
+  LLVMJITModule module("Test Module");
+  JITTuple layer1(JITTypeTag::TUPLE, JITBackendTag::LLVMJIT);
```

Comment:
Maybe we could add a virtual method ```createJITTuple``` in ```JITFunction```, so we needn't to pass these tags.

Reply:
get it